lib/screens/authenticate/verified_email_check.dart

```diff
@@ -70,6 +70,13 @@ class _VerificationCheckState extends State<VerificationCheck> {
                     } catch (e) {
                       setState(() {
                         error = e.message;
+                        Scaffold.of(context).showSnackBar(SnackBar(
+                            backgroundColor: Theme.of(context).primaryColor,
+                            content: Text(
+                              e.toString(),
```

Comment:
Use e.message here or custom string according to error. e.toString() gives were a complicated string of error object hard to understand for a normal user.